pithos-save.py

```diff
@@ -16,38 +16,45 @@
 
 from pithos.plugin import PithosPlugin
 from pithos.pithosconfig import get_data_file
-from mutagen.mp4 import MP4
 import os
 from shutil import copyfile as copy
-from glob import glob
+import urllib
+import eyed3
+import eyed3.id3
 
 class SavePlugin(PithosPlugin):
     preference = 'save'
-    _MUSIC_DIR = os.getenv("XDG_MUSIC_DIR")
+    _MUSIC_DIR = os.getenv("HOME") + '/Music/'
 
     def on_enable(self):
-        self.song_rating_changed_handle = self.window.connect('song-rating-changed', self.song_rating_changed)
-        
+        self.song_rating_changed_handle = self.window.connect('song-rating-changed', self.song_rating_changed) 
+
     def song_rating_changed(self, window,  song):
         if song.rating and song.rating_str == 'love':
-            tmp_files = glob("/tmp/pithos-??????")
-            if not tmp_files:
-                return
-            src_file = tmp_files[-1]
+            fetch_file = '/tmp/pithos-fetch'
```

Comment:
Yea looks like gstreamer no longer has a temp file this I created one, I will do another pull with the logic being better.